LayoutTests/fast/dom/NodeIterator/NodeIterator-leak-document-expected.txt

```diff
@@ -0,0 +1,5 @@
+PASS endingDocumentCount - startingDocumentCount < testingCount * 0.8 is true
```

Comment:
Would be nice for have a `description();` in all these tests.

Reply:
Good point.